### DISCUSSION TITLE
unleash-client-java 8.3.0 seems like an empty release?

### DISCUSSION THREAD

**Discussion by paulrvvup** [Unleash/unleash#4582] [Q&A] [answered]
It looks like the diff for the 8.3.0 release is empty: https://github.com/Unleash/unleash-client-java/compare/unleash-client-java-8.2.1...unleash-client-java-8.3.0. Is there a reason this was released, or am I missing something? Just wanted to make sure this was intended. Thanks!

**Comment by gardleopard** [comment, marked as answer]
Hi @paulrvvup,

Sorry for the late response :( Yes, that release was empty. Its out there now, but you should use the 8.3.1 release instead as that has an important bugfix included.
